fem/tmop.hpp

```diff
@@ -1486,6 +1568,13 @@ class TMOP_Integrator : public NonlinearFormIntegrator
                                     ElementTransformation &T,
                                     const Vector &elfun, DenseMatrix &elmat);
 
+   TMOP_QualityMetric &GetAMRQualityMetric() { return *h_metric; }
+
+   void Update();
```

Comment:
Should we name this `UpdateAfterMeshTopologyChange` to correspond to the other update function, and the function in `DiscreteTC`?